napalm_logs/config/junos/OSPF_NEIGHBOR_DOWN.yml

```diff
@@ -0,0 +1,36 @@
+# This error tag corresponds to syslog messages notifying that the configured
+# ospf neighbor has changed state from Full
+messages:
+  # 'error' should be unique and vendor agnostic. Currently we are using the JUNOS syslog message name as the canonical name.
+  # This may change if we are able to find a more well defined naming system.
+  - error: OSPF_NEIGHBOR_DOWN
+    tag: RPD_OSPF_NBRDOWN
+    values:
+      neighbor: ([\w\d:\.]+)
+      interface: ([\w\-\/\.\d]+)
+      area: (\d+\.\d+\.\d+\.\d+)
+      reason_message: ([\w ]+)
```

Comment:
You should not use special characters in the key name as we escape all special characters. This will cause a key error when parsing.

Reply:
I mean `reason_message`, it should be `reasonMessage`